src/klein/_isession.py

```diff
@@ -2,10 +2,7 @@
 
 import attr
 
-try:
-    from constantly import NamedConstant, Names
-except ImportError:  # pragma: no cover
-    from twisted.python.constants import NamedConstant, Names
```

Comment:
`pragma: no cover` is here because no version of Twisted in our test matrix causes this code to be used. Let's just assume Twisted is using `constantly`.